build/stf-run-ci/tasks/clone_repos.yml

```diff
@@ -58,3 +58,17 @@
         repo: https://github.com/infrawatch/prometheus-webhook-snmp
         dest: working/prometheus-webhook-snmp
         version: master
+
+- name: Get Loki Operator
+  block:
+    - name: Try cloning same-named branch or override branch
+      git:
+        repo: https://github.com/viaq/loki-operator
+        dest: working/loki-operator
+        version: "{{ loki_operator_branch | default(branch, true) }}"
```

Comment:
Note that this branch logic isn't going to work quite like the other components because in general if we were to make a  PR against the loki-operator, it will not be a branch on the viaq repo since we don't have write access to it. It still might be useful, and it's not dangerous, but I wanted to point this out.

Reply:
Might be worth just adding a little comment in the file about that. I suspect it's generally fine just because I'm not sure how much we'll be doing PRs and needing to do tests against it, but obviously that won't work with this repo.

Reply:
I added a note about this to the file